Frontend/CO.CDP.OrganisationApp/Pages/Registration/JoinOrganisation.cshtml

```diff
@@ -0,0 +1,102 @@
+                    <div class="govuk-summary-list__row">
+                        <dt class="govuk-summary-list__key">
+                            Organisation name
+                        </dt>
+                        <dd class="govuk-summary-list__value">
+                            <p class="govuk-body">@organisationDetails.Name</p>
+                        </dd>
+                    </div>
+                </dl>
+                <br/>
+                <h2 class="govuk-heading-m">User details</h2>
+                <dl class="govuk-summary-list">
+                    <div class="govuk-summary-list__row">
+                        <dt class="govuk-summary-list__key">
+                            Firstname
+                        </dt>
+                        <dd class="govuk-summary-list__value">
+                            <p class="govuk-body">John</p>
+                        </dd>
+                    </div>
+                    <div class="govuk-summary-list__row">
+                        <dt class="govuk-summary-list__key">
+                            Lastname
+                        </dt>
+                        <dd class="govuk-summary-list__value">
+                            <p class="govuk-body">Johnson</p>
+                        </dd>
+                    </div>
+                    <div class="govuk-summary-list__row">
+                        <dt class="govuk-summary-list__key">
+                            Email address
+                        </dt>
+                        <dd class="govuk-summary-list__value">
+                            <p class="govuk-body">john@johnson.com</p>
+                        </dd>
+                    </div>
+                </dl>
+            </fieldset>
+        </div>
+    </div>
+    <br/>
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <form method="post">
+                <fieldset class="govuk-fieldset" data-required="" data-error="Select yes if you want to join this organisation" aria-required="true">
+                    <div class="govuk-form-group @(joinHasError ? "govuk-form-group--error" : "")">
+                        <h2 class="govuk-heading-m">
+                            <span class="govuk-label--m">Would you like to join @organisationDetails.Name?</span>
+                        </h2>
+                        @if (joinHasError)
+                        {
+                            <p id="approval-error" class="govuk-error-message">
+                                <span class="govuk-visually-hidden">Error:</span>
+                                @Html.ValidationMessageFor(m => m.Join)
+                            </p>
+                        }
+                        <div class="govuk-radios govuk-radios--inline" data-module="govuk-radios">
+                            <div class="govuk-radios__item">
+                                <input class="govuk-radios__input" id="join-YES" name="Join" type="radio"
```

Comment:
The id of this needs to be `nameof(Model.Join)` because when errors are raised, the error summary will link to that. The id for the one below it can then be `nameof(Model.Join)-1`. As it stands at the moment, the error summary will not link anywhere

Reply:
while you're in this area, also best to have `name="@nameof(Model.Join)"` because then we get compile time safety of knowing that the input matches the name in the Model